packages/babel-plugin-material-ui/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@material-ui/babel-plugin-material-ui",
```

Comment:
`babel-plugin-material-ui` is already in use.
Project is not maintained anymore and has been deprecated on npm
https://www.npmjs.com/package/babel-plugin-material-ui